Storybook build error : storybook-chromatic not resolved

<!---
Thanks for filing an issue 😄! Before you submit, please read the following:

Search open/closed issues before submitting since someone might have asked the same thing before!
-->

# 🐛 Bug Report 

I did a fresh clone and install of the repo per the contributing guide, and came across this error:

```
ERROR in ./.storybook/config.js
Module not found: Error: Can't resolve 'storybook-chromatic' in '/Users/baird/code/repos/react-spectrum/.storybook'
```

https://www.npmjs.com/package/storybook-chromatic suggests this package has been deprecated in favor of 'chromatic'

## 🤔 Expected Behavior

Storybook should build properly with no errors

## 😯 Current Behavior

<!--- Tell us what happens instead of the expected behavior. Please include a link to the storybook if possible. -->

<!--- If you are seeing an error, please include the full error message and stack trace -->

## 💁 Possible Solution

<!--- Not obligatory, but suggest a fix/reason for the bug -->

## 🔦 Context

<!--- How has this issue affected you? What are you trying to accomplish? -->

<!--- Providing context helps us come up with a solution that is most useful in the real world -->

## 💻 Code Sample

<!-- Please provide a code repository, gist, code snippet or sample files to reproduce the issue -->

## 🌍 Your Environment

<!--- Include as many relevant details about the environment you experienced the bug in -->

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   |
| Browser          |
| Operating System |

## 🧢 Your Company/Team

Analytics

## 🕷 Tracking Issue (optional)

<!--- Link to the issue in your system, if publicly available -->

--- a/.chromatic/config.js
+++ b/.chromatic/config.js
@@ -1,4 +1,4 @@
-import 'storybook-chromatic';
+import 'chromatic';
 import {configure, addDecorator, addParameters} from '@storybook/react';
 import {configureActions} from '@storybook/addon-actions';
 import React from 'react';

--- a/.storybook/config.js
+++ b/.storybook/config.js
@@ -1,4 +1,3 @@
-import 'chromatic';
 import {configure, addDecorator, addParameters} from '@storybook/react';
 import {configureActions} from '@storybook/addon-actions';
 import React from 'react';

--- a/.storybook/config.js
+++ b/.storybook/config.js
@@ -1,4 +1,4 @@
-import 'storybook-chromatic';
+import 'chromatic';
 import {configure, addDecorator, addParameters} from '@storybook/react';
 import {configureActions} from '@storybook/addon-actions';
 import React from 'react';

